.changeset/old-llamas-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@studiocms/devapps": patch
```

Comment:
would it make more sense to change the name of the package to something a bit more self explanatory like `devtoolbarapps`?

Reply:
Idk, I feel these _are_ dev toolbar apps... But I guess not _exclusively_ dev toolbar apps. In this case it's an app that has been shoved into the dev toolbar... That being said, I don't like the name `devapps` but don't have any better suggestions off the top of my head

Reply:
Not 100% specific to just being devtoolbar apps.  for example the current one also has a page available at `/_studiocms-devapps/libsql-viewer`